plugins/processors/t128_pass/README.md

```diff
@@ -98,3 +108,24 @@ measurement tag1=value1 1612214810000000000
 measurement tag1=value2 1612214810000000000
 - measurement tag1=value3 1612214810000000000
 ```
+
+### Basic Field Example Filter:
+
+Fields can also be filtered on by themselves or AND'd together with tags.
+
+```toml
+[[processors.t128_filter]]
+  [[processors.t128_filter.condition]]
+
+  [processors.t128_filter.condition.tags]
+    tag1 = ["value1"]
+
+  [processors.t128_filter.condition.fields.string]
```

Comment:
I think this works alright and allows for future proofing. I guess my original idea was to have them just be `interface{}` and then go through a list of type assertions until I found one that worked, but I was dreading how sketchy that was going to be. I like this because of the explicitness of it.

I kept the string conversion on all types because I couldn't see a downside of it.